workspace/resource_notebook.go

```diff
@@ -205,10 +216,14 @@ func getEnvAsInt(envName string, defaultValue int) int {
 	return defaultValue
 }
 
-func (a NotebooksAPI) ListParallel(path string, recursive bool) ([]ObjectStatus, error) {
```

Comment:
Hm, where was this parameter used before?

Reply:
Really it wasn't used except for "compatibility" with the non-parallel implementation. But it makes no sense to use parallel implementation for non-recursive listing.